drivers/storage/portworx/util/csi_generator.go

```diff
@@ -260,9 +260,9 @@ func (c *CSIConfiguration) DriverBasePath() string {
 }
 
 func (g *CSIGenerator) getSidecarContainerVersionsV1_0() *CSIImages {
-	provisionerImage := "k8s.gcr.io/sig-storage/csi-provisioner:v3.3.0"
-	snapshotterImage := "k8s.gcr.io/sig-storage/csi-snapshotter:v6.1.0"
-	snapshotControllerImage := "k8s.gcr.io/sig-storage/snapshot-controller:v6.1.0"
+	provisionerImage := "registry.k8s.io/sig-storage/csi-provisioner:v3.3.0"
```

Comment:
Should this use the same constant as above? Replace with a `fmt.Sprintf(...`?

Reply:
sure, i can make the optimization, was just doing a string replacement in the code.